harness/determined/pytorch/_pytorch_context.py

```diff
@@ -129,6 +129,12 @@ def get_lr_scheduler(self) -> Optional[pytorch.LRScheduler]:
             return self.lr_schedulers[0]
         return None
 
+    def get_use_amp(self) -> bool:
+        return self._use_amp
```

Comment:
The PyTorchTrialContext is primarily about exposing methods to users that they can use while training, but this is an internal implementation detail, so we need to either make it a "private" method (`_get_use_amp()`) or just call `context._use_amp` directly in the `PyTorchTrialController`.

Personally I don't care which, but we can't leak internal details into the public API.